datafusion-cli/src/command.rs

```diff
@@ -117,6 +121,7 @@ impl FromStr for Command {
             ("d", None) => Self::ListTables,
             ("d", Some(name)) => Self::DescribeTable(name.into()),
             ("?", None) => Self::Help,
+            ("quiet", Some(b)) => Self::QuietMode(b == "true" || b == "t"),
```

Comment:
In the doc, we just accept true/false, i think this code is inconsistent with the doc.
Maybe we should consider the lower/upper case.

Reply:
You're right, in psql it's common to have on|off flags (case insensitive), should we do the same ?
```
\x [on|off|auto]       toggle expanded output (currently off)
```